doc/install.rst

```diff
@@ -108,6 +108,7 @@ The following are optional dependencies:
 * `IPython <https://ipython.org>`__: For embedding the figures in Jupyter notebooks (recommended).
 * `Contextily <https://contextily.readthedocs.io>`__: For retrieving tile maps from the internet.
 * `GeoPandas <https://geopandas.org>`__: For using and plotting GeoDataFrame objects.
+* `RioXarray <https://https://corteva.github.io/rioxarray>`__: For saving multi-band rasters to GeoTIFFs.
```

Comment:
```suggestion
* `RioXarray <https://corteva.github.io/rioxarray>`__: For saving multi-band rasters to GeoTIFFs.
```

Reply:
Good catch, typo fixed in fa8641b12ff77fed86653426e78b75157dcdf223.